packages/flutter_tools/test/integration.shard/multidex_build_test.dart

```diff
@@ -52,8 +51,8 @@ void main() {
       '--debug',
     ], workingDirectory: tempDir.path);
 
+    expect(result, const ProcessResultMatcher(exitCode: 1));
     expect(result.stderr.toString(), contains('Cannot fit requested classes in a single dex file'));
     expect(result.stderr.toString(), contains('The number of method references in a .dex file cannot exceed 64K.'));
```

Comment:
Could we refactor this to use the matcher's `stdout` and `stderr` matchers?

Reply:
these are both on stderr though

Reply:
Ah I was going too fast, makes sense though. LGTM on everything else then

Reply:
I briefly thought about some kind of API that would allow either a pattern OR `Iterable<Pattern>` for stdout matchers, but then thought better of it given how confusing the `process.stdout` API is (it's typed `dynamic` but is one of `String` or `List<int>`